### ISSUE TITLE
Define new projectile prefabs

### ISSUE THREAD

**Issue by sdiaz7** [NestorTejero/ES2016A#170]
### Definition of Done:
- At least 3 types of Projectile Prefabs
- All models correctly orientated with the Z axis looking forward
- Linked onto the Enemy Prefabs on the `projectilePrefab` field
- Each has own properties such as gravity, and parabollic trajectory

**Estimated 4 Hours**

**Comment by Nerucius** [comment]
**Status Report**
- Updated DoD
- Fundamental changes to how projectile prefabs are handled made on #162 
- Waiting on @vmganela for correct axis on projectiles
  - Related Pull request at #156 

**Estimated 2 Hours**

**Comment by Nerucius** [comment]
**New Commit b268ec6a5b62ce43de84d2ab18bcc20ecfee189d on branch [dev_Issue170](https://github.com/NestorTejero/ES2016A/tree/dev_Issue170):**
- Created 3 new prefabs for projectiles
- Created all necessary materials

![capture](https://cloud.githubusercontent.com/assets/7935504/20908905/6ada1504-bb58-11e6-817a-856f71b0b741.PNG)

**New estimate 1 Hour to completion**

**Comment by Nerucius** [comment]
Pull request at #184

**Comment by sdiaz7** [comment]
Implementation satisfies the definition of done, and has been merged into DEV. I proceed to close the issue and move it to DONE.
